### ISSUE TITLE
About TODO #94

### ISSUE THREAD

**Issue by loozyoni** [pinchbv/floor#731]
I update table row using "update tableName set ..." and it did not notify the stream,
and I checked and see the comment:
    // TODO differentiate between different query kinds (select, update, delete, insert)
    // TODO #94 differentiate between different query kinds (select, update, delete, insert)
    //  this enables to notify the observers
    //  also requires extracting the table name :(

My question:
When do you think it will be ready?

Thanks

**Comment by dkaera** [comment]
thanks for raising this issue. I think it's a good point to do next 😄

**Comment by loozyoni** [comment]
Thanks for your fast reply.
We look forward to it.🙂

**Comment by drEfrati** [comment]
Our team also encounter this problem and it would be great to see it in the next version.
Thanks.

**Comment by dkaera** [comment]
Yes, I'm sure I can make some time this month for implementation.

**Comment by dkaera** [comment]
Many thanks to Simon Binder for the SQL parser 🙇‍♂️
It saved a lot of time to implement this feature.
Here is PR #733
